components/landing/banner/Banner.module.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../styles/globals.scss';
+
```

Comment:
see comment below about the name of these files.

Reply:
Banner was renamed to Testimonials.